cmd/kaf/kaf.go

```diff
@@ -29,6 +29,41 @@ func getConfig() (saramaConfig *sarama.Config) {
 		saramaConfig.Net.SASL.User = cluster.SASL.Username
 		saramaConfig.Net.SASL.Password = cluster.SASL.Password
 	}
+	if cluster.TLS != nil && cluster.SecurityProtocol != "SASL_SSL" {
+		saramaConfig.Net.TLS.Enable = true
+		tlsConfig := &tls.Config{
+			InsecureSkipVerify: cluster.TLS.Insecure,
+		}
+
+		caCert, err := ioutil.ReadFile(cluster.TLS.Cafile)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+		caCertPool := x509.NewCertPool()
+		caCertPool.AppendCertsFromPEM(caCert)
+		tlsConfig.RootCAs = caCertPool
+
+		clientCert, err := ioutil.ReadFile(cluster.TLS.Clientfile)
```

Comment:
i never used kafka with TLS tbh. i wonder; in TLS client certs are optional, so we could make the client cert pair optional.

are there cases where the kafka server will not require client auth (no client cert enforced), but runs with TLS ?

Reply:
> are there cases where the kafka server will not require client auth (no client cert enforced), but runs with TLS ?

Not to my knowledge. If the cluster requires TLS for client connections, we will need all three components. For what it's worth, I'm proposing this change to enable client connections to [Heroku Kafka](https://devcenter.heroku.com/articles/kafka-on-heroku#connecting-to-a-kafka-cluster).

Reply:
fine for me - we can still enable such a use case later on if it's really needed by someone.